doc/nvim-tree-lua.txt

```diff
@@ -859,6 +904,11 @@ UI rendering setup
             Glyph for symlinks to files.
               Type: `string`, Default: `""`
 
+            *nvim-tree.renderer.icons.glyphs.modified*
+            Icon to display for modified files. Can be multiple characters if
+            |renderer.icons.modified_placement| isn't `"signcolumn"`.
```

Comment:
That's quite nice. I dont't think `get_padding` will handle multiple characters before, so we should restrict multiple characters to "after" only.

It would be consistent with vim's [+] after which I imagine is the use case.

Reply:
Hmmmm, with my testing `get_padding` only assume `renderer.indent_markers.icons` is a single character? everything else seems to be fine with multiple character.
This is on `master`:
![image](https://user-images.githubusercontent.com/38484873/209271149-e42b8606-3d29-4048-bc6a-ca71929029ff.png)
I did see some code assuming ` renderer.icons.padding` is a single character though. Though my next commit will make that no longer the case due to some refactoring.

Reply:
Update: upon further testing, it looks like we assume all icons that could show up in the sign column are at most 2 characters. This is because `vim.fn.sign_define()` doesn't allow more than two characters. We can fix this in another PR by only calling `vim.fn.sign_define()` when placement is actually `"signcolumn"`.